src/legacy/core_plugins/kibana/public/dashboard/listing/dashboard_listing.test.js

```diff
@@ -149,6 +149,25 @@ describe('after fetch', () => {
     expect(component).toMatchSnapshot();
   });
 
+  test('hideWriteControls with dashboards', async () => {
+    const component = shallowWithIntl(<DashboardListing.WrappedComponent
+      findItems={find.bind(null, 1)}
+      deleteItems={() => {}}
+      createItem={() => {}}
+      editItem={() => {}}
+      getViewUrl={() => {}}
+      listingLimit={1}
+      hideWriteControls={true}
+    />);
+
+    // Ensure all promises resolve
+    await new Promise(resolve => process.nextTick(resolve));
+    // Ensure the state changes are reflected
+    component.update();
+
+    expect(component).toMatchSnapshot();
+  });
```

Comment:
nit: This probably has more to do with the way the tests were originally written than with the ones you've added, but I noticed there's no difference in the snapshot output between `hideWriteControls with no dashboards` and `hideWriteControls with dashboards`... They're both identical, which makes me suspect they probably aren't that useful.

Reply:
Good call, this test was written prior to the consolidation that Liza did to use the `TableListView`, I will remove the duplicate.